pkg/odo/cli/component/update.go

```diff
@@ -134,7 +165,14 @@ func (uo *UpdateOptions) Run() (err error) {
 		if err = component.Update(uo.Context.Client, *uo.cmpConfig, cmpPath, stdout); err != nil {
 			return err
 		}
-		log.Successf("The component %s was updated successfully, please use 'odo push' to push your local changes", uo.componentName)
+		if uo.now {
```

Comment:
same as above https://github.com/openshift/odo/pull/1937/files#r311460775. 
This would help us DRY the code base a bit and make it more readable